apache/app/demonstrator.py

```diff
@@ -50,25 +17,29 @@ def delivery_report(err, msg):
     else:
         print('Message delivered to {} [{}]'.format(msg.topic(), msg.partition()))
 
-class Demonstrator:
-    value_schema = avro.loads(value_schema_str)
-    key_schema = avro.loads(key_schema_str)
 
+class Demonstrator:
 
-    def __init__(self, broker_url, registry_url):
+    def __init__(self, broker_url, registry_url, topic):
         self.broker_url = broker_url
         self.registry_url = registry_url
 
+        if topic == "login":
+            self.value_schema = schema_login
+            self.key_schema = schema_login
+        if topic == "message":
+            self.value_schema = schema_message
+            self.key_schema = schema_message
+        if topic == "mouse":
+            self.value_schema = schema_mouse
+            self.key_schema = schema_mouse
```

Comment:
Dafür wäre das "switch-case" Statement etwas besser geeignet. Noch besser wäre, das in eine Methode auszulagern, weil wir quasi dreimal das gleiche machen :)